crates/router/src/connector/stripe/transformers.rs

```diff
@@ -1110,11 +1115,36 @@ impl TryFrom<&payments::BankRedirectData> for StripeBillingAddress {
             }),
             payments::BankRedirectData::Ideal {
                 billing_details, ..
-            } => Ok(Self {
-                name: billing_details.billing_name.clone(),
-                email: billing_details.email.clone(),
-                ..Self::default()
-            }),
+            } => {
+                let billing_name = billing_details
+                    .clone()
+                    .and_then(|billing_data| billing_data.billing_name.clone());
+
+                let billing_email = billing_details
+                    .clone()
+                    .and_then(|billing_data| billing_data.email.clone());
+                match is_merchant_initiated_mandate_payment {
```

Comment:
is this mandatory only incase of MIT and not in CIT or Normal transaction?

Reply:
yes, this field only needed during CIT